tests/validation/tests/v3_api/test_airgap_ha.py

```diff
@@ -377,6 +385,24 @@ def setup_airgap_rancher(bastion_node, number_of_nodes=NUMBER_OF_INSTANCES):
     bastion_node.execute_command(
         kube_config+"/snap/bin/kubectl create namespace cattle-system && 10")
     create_tls_secrets(bastion_node)
+    bootstrap_password=""
+    if RANCHER_HELM_EXTRA_SETTINGS:
+        for setting in RANCHER_HELM_EXTRA_SETTINGS.split("--set "):
+            settings_keypair = RANCHER_HELM_EXTRA_SETTINGS.split("=")
+            if "bootstrapPassword" in settings_keypair[0]:
```

Comment:
If the bootstrap password is known, does this mean that the `setup_rancher_server` function should be refactored too?

So here setting the `bootstrap_password` manually is required in this logic what would happen if `RANCHER_HELM_EXTRA_SETTINGS` is empty.

Reply:
So we're default this flag in the airgap job much like the rancher ha. Is that correct ?

Reply:
that's correct, there's a supporting jenkins-job PR as well.